velox/functions/lib/TimeUtils.h

```diff
@@ -92,6 +94,23 @@ FOLLY_ALWAYS_INLINE int32_t getDayOfYear(const std::tm& time) {
   return time.tm_yday + 1;
 }
 
+FOLLY_ALWAYS_INLINE unsigned getWeek(
```

Comment:
nit: could we use something like uint32_t to make the type length more explicit?